backend/src/openarchiefbeheer/conf/dev.py

```diff
@@ -92,6 +94,17 @@
     r"django\.db\.models\.fields",
 )
 
+# End-to-end test settings
+E2E_PORT = config("E2E_PORT", default=8080)
```

Comment:
Maybe E2E_PORT and SERVE_FRONTEND can be put next to each other, since they both have to do with E2E tests? And maybe  SERVE_FRONTEND can become E2E_SERVE_FRONTEND :thinking: